test/test_image_streamer_build_plan.py

```diff
@@ -34,9 +62,9 @@ class TestBuildPlanModule(ImageStreamerBaseTest):
     @pytest.fixture(autouse=True)
     def specific_set_up(self):
         # Load scenarios from module examples
-        self.BUILD_PLAN_CREATE = self.EXAMPLES[0]['image_streamer_build_plan']
-        self.BUILD_PLAN_UPDATE = self.EXAMPLES[1]['image_streamer_build_plan']
-        self.BUILD_PLAN_DELETE = self.EXAMPLES[2]['image_streamer_build_plan']
+        self.BUILD_PLAN_CREATE = BUILD_PLAN_CREATE
```

Comment:
Is it really required to create instance variables using the global variables?
I think you can use the global variables itself wherever required.